blockchain/tx_pool_test.go

```diff
@@ -1303,6 +1312,62 @@ func TestTransactionPendingGlobalLimiting(t *testing.T) {
 	}
 }
 
+// Test the limit on transaction size is enforced correctly.
+// This test verifies every transaction having allowed size
+// is added to the pool, and longer transactions are rejected.
+func TestTransactionAllowedTxSize(t *testing.T) {
```

Comment:
Can we make this test more readable? e.g.,
```go
	testcases := []struct {
		tx      *types.Transaction
		success bool
	}{
		{ // maximal allowed size
			pricedDataTransaction(0, 100000000, big.NewInt(1), key, dataSize),
			true,
		},
		// ...
		{ // random not allowed size
			pricedDataTransaction(2, 100000000, big.NewInt(1), key, dataSize+1+uint64(rand.Intn(int(10*MaxTxDataSize)))),
			false,
		},
		// ...
	}
```

Reply:
It looks better, thanks :) 

I applied at https://github.com/klaytn/klaytn/pull/1733/commits/718da50149264f5babf510525074b4ca3a2ae5e9. Please take a look.

Reply:
I think the second tc is wrong (it's not random).

Also, I thought "a transaction of minimal not allowed size" has the size of either `dataSize + 1` or `dataSize + txSlotSize`, since "a transaction with maximal allowed size" has the size of `dataSize`. Am I missing something??

Reply:
>I think the second tc is wrong (it's not random)

it was my mistake.. i fixed it thanks. https://github.com/klaytn/klaytn/pull/1733/commits/a1b1a6188be1d0a2e20be6fd2562f4363b33c719

You're right. Is something wrong?

Reply:
Revised the comment. https://github.com/klaytn/klaytn/pull/1733/commits/c81cf24aeb94689e04b281e6580a08c39234a286